roles/ci_gen_kustomize_values/tasks/edpm_core_facts.yml

```diff
@@ -0,0 +1,37 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Fetch the already deployed services for further usage
+  when: ci_gen_kustomize_fetch_ocp_state
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command:
```

Comment:
can't we use `kubernetes.core.k8sinfo` instead?

Reply:
Short answer: no
Long answer: yes, but, it's a mess, why? API Versions. The dumb `kubernetes.core.k8sinfo` is not able to properly guess custom CRs versions, so we need to explicitly pass them, meaning hard-coding it. I don't want to hard-code that info in the framework, cause a change in the API Version of the CR is going to need a CI-framework change.